Dockerfile

```diff
@@ -22,13 +26,14 @@ RUN \
       go build -o /sshpiperd/plugins -tags "${BUILDTAGS}" ./plugin/... ./e2e/testplugin/...
     fi
 HEREDOC
-ADD entrypoint.sh /sshpiperd
```

Comment:
The E2E test failure is due to `testrunner` relying on `entrypoint.sh` from the `builder` stage (_but there was nothing in the `Dockerfile` to suggest that expectation_):

https://github.com/tg123/sshpiper/blob/52e8f9afc4e3c7e4e0c558b6857bd2b803d0bbff/e2e/e2eentry.sh#L4-L5

This has been resolved in: https://github.com/tg123/sshpiper/pull/575

This PR will be updated once that PR is merged (_which has a **breaking change** in that it **removes** the [docker specific `PLUGIN` ENV](https://github.com/tg123/sshpiper/pull/575#issue-3026325888)_).